### DISCUSSION TITLE
zoom in android tv view

### DISCUSSION THREAD

**Discussion by rickysarraf** [Radiokot/photoprism-android-client#78] [General]
Now that this app has progressed enough that some of us are already side-loading it to our TVs. Overall, the basic functionality is there and working fine.

Would be nice to have:

* Zoom button in TV view
* Rotate button in TV view
* Like button

**Comment by Radiokot** [comment]
How do you zoom with a remote control?

**Reply by rickysarraf** [reply]
So, in the current implementation, on the remote `left` and `right` are used for navigation, in picture view mode. How about using `up` and `down` for zoom ?

For rotate, I guess you could add one more overlay icon named `More`. And push moar such features under that button. I guess it'd then  cover `like` as well.

**Reply by rickysarraf** [reply]
On the other hand, if remote buttons are all used already, how about adding a soft button on the overlay with specific action. 

Like:

* Zoom button: When pressed `up` and `down` behave as zoom in that context
* Rotate button: When pressed `left` and `right` behave as rotate left/right in that context.

**Reply by Radiokot** [reply]
Thanks for describing your ideas.
I don't see the TV-specific features are coming in the nearest future. Maybe I don't get something, but for me, it is currently a pain to implement, test and fix the keyboard navigation on a TV, I'm glad that there is a bare minimum of browsing 😄

**Reply by rickysarraf** [reply]
Such projects are supposed to be fun. So please don't stress yourself. You may consider it when you are ready to use it on your TV. :-)

Until then, what you've already created is good enough. 🙏🏽
